### PR TITLE
Stable Diffusion XL

### PR DESCRIPTION
**Pull request by msaroufim**

### CODE REVIEW

**Review by cpuhrsch — APPROVED** [review]

File: torchbenchmark/models/stable_diffusion_xl/__init__.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    DEFAULT_EVAL_CUDA_PRECISION = "fp16"
+
+    def __init__(self, test, device, batch_size=None, extra_args=[]):
+        HuggingFaceAuthMixin.__init__(self)
+        super().__init__(test=test, device=device,
+                         batch_size=batch_size, extra_args=extra_args)
+        model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+        self.pipe = DiffusionPipeline.from_pretrained(model_id).to(device)
+        self.list_of_inputs = [
+            torch.randn(1, 4, 256, 256).to(self.device),
```
Comment (xuzhao9) [inline torchbenchmark/models/stable_diffusion_xl/__init__.py:30]: Question: do we need to convert the tensor precision to `fp16`? Or does the model work on fp16 by default?

**Review by xuzhao9 — APPROVED** [review]
LGTM with minor comments

File: torchbenchmark/models/stable_diffusion_xl/__init__.py (hunk 1 above)
Comment (msaroufim) [reply torchbenchmark/models/stable_diffusion_xl/__init__.py:30]: It's just a reasonable default, I can remove this

**Comment by msaroufim** [comment]
@xuzhao9 I moved this model to canary since it OOMs on an A10G

**Pull request closed without merging**